controllers/v1/deletionAuditLogs.js

```
@@ -0,0 +1,28 @@
+/**
+ * name : deletionAuditLogs.js.
+ * author : Mallanagouda R Biradar
+ * created-date : 25-July-2025
+ * Description : deletionAuditLogs related information.
+ */
+
+module.exports = class deletionAuditLogs extends Abstract {
```

ESLint

[error] 'Abstract' is not defined.

(no-undef)